README.md

```
@@ -160,8 +160,23 @@ Lightspeed Core Stack is based on the FastAPI framework (Uvicorn). The service i
-Installation steps depends on operation system. Please look at instructions for your system:
+## Clone the Repository
+
+Lightspeed Stack uses a git submodule for external providers (required for features like OKP RAG). Clone with submodules:
+
+```bash
+git clone --recursive https://github.com/lightspeed-core/lightspeed-stack.git
+cd lightspeed-stack
+```
 
+**If you already cloned without `--recursive`:**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)